code/game/objects/buckling.dm

```diff
@@ -1,6 +1,9 @@
+#define NO_BUCKLE_LYING -1
```

Comment:
I'm uncomfortable putting a define that multiple files use in a non-__DEFINES area. Can you move this out and document tit?

Reply:
Sure